### PR TITLE
feat(stickers): add stickers

### PR DESCRIPTION
**Pull request by verarojman**
Related to:
- https://github.com/Platoniq/journal-content/pull/6
- https://github.com/Platoniq/jekyll-plugin-platoniq-journal/pull/16

image
image

image

### CODE REVIEW

File: _sass/partials/_grid.scss [hunk 1]
```diff
@@ -255,7 +255,7 @@
   // }
 
   .container{
-    max-width: $maxwidht-container;
```
Comment (agustibr) [inline _sass/partials/_grid.scss:258]: 😵‍💫

File: scripts/create_sticker_collection.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env/ruby
+# frozen_string_literal: true
+
+#
+# This ruby script is to create the front matter files which contain
```
Comment (agustibr) [inline scripts/create_sticker_collection.rb:5]: 🤩

**Review by agustibr — APPROVED** [review]
Nice!! 🎨

File: _includes/default/stickers.liquid [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% assign stickers = page.stickers %}
+{% assign base_path = page.stickers %}
```
[the remark this answers is not in the document]
Comment (verarojman) [reply _includes/default/stickers.liquid:2]: Hahaha totally missed this

**Comment by verarojman** [comment]
> warning Oops some things broke with the suggested changes, let me check it before merging @agustibr

✅ Fixed via #35 and https://github.com/Platoniq/journal-content/pull/7

**Pull request merged**